src/page-modules/assistant/server/journey-planner/validators.ts

```diff
@@ -146,6 +146,10 @@ export const tripPatternWithDetailsSchema = z.object({
   ),
 });
 
+export const linesSchema = z.object({
+  publicCodeLineMapString: z.string(),
```

Comment:
Given previous suggestion this needs to be updated to reflect actual type
```suggestion
  publicCodeLineMapString: z.record(z.string(), z.array(z.string())),
```